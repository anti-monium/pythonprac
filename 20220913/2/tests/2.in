[0,-7,8,329,82,-23444442,2894321]
