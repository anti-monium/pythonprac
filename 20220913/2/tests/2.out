-23444442, -7, 0, 8, 82, 329, 2894321
